mutters-core/src/main/java/com/rabidgremlin/mutters/core/Intent.java

```diff
@@ -1,17 +1,22 @@
+/* Licensed under Apache-2.0 */
 package com.rabidgremlin.mutters.core;
 
 import java.util.Collection;
 import java.util.Collections;
+import java.util.Objects;
 
 /**
- * This class represents an intent. Each intent has a unique name and zero or more Slots that are used to extract
- * entities out of a user's input.
+ * This class represents an intent. Each intent has a unique name and zero or
+ * more Slots that are used to extract entities out of a user's input.
  * 
  * @author rabidgremlin
  *
  */
 public class Intent
```

Comment:
maybe this class needs equals/hashcode/toString (just on `name`)?

Reply:
mmm, dangerous territory as some might expect the slots to also have to match for an intent to be equal...

Reply:
kk